addons/web/static/src/js/views/calendar/calendar_view.js

```diff
@@ -195,7 +195,8 @@ var CalendarView = AbstractView.extend({
         this.loadParams.filters = filters;
         this.loadParams.mode = (params.context && params.context.default_mode) || attrs.mode;
         this.loadParams.scales = scales;
-        this.loadParams.initialDate = moment(params.initialDate || new Date());
+        this.loadParams.initialDate = moment((params.context && params.context.initial_date) ||
```

Comment:
Linting: is it multi-line-guidelines-compliant ? Shouldn't we have something like
```
moment(
    (params.context && params.context.initial_date) ||
    params.initialDate ||
    new Date()
);